integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -0,0 +1,66 @@
+import typing as tp
+import allure
+import json
+from jsonschema import Draft4Validator
+import pytest
+from integration.tests.basic.helpers.basic import BaseMixin
+from utils.helpers import wait_condition
+
+SCHEMAS = "./integration/tests/tracer/schemas/"
+
+
+@allure.story("Tracer API RPC calls debug methods check")
+class TestTracerRpcCalls(BaseMixin):
```

Comment:
Change class name because it the same as class second file

Reply:
done